tee-worker/local-setup/launch.py

```diff
@@ -79,6 +80,14 @@ def is_port_open(port):
     except OSError:
         return False
 
+def is_directory_accessible(directory_path):
+    if os.path.exists(directory_path):
```

Comment:
I think it's better to simply reassign the directory as long as it exists.

Because even if you can access it (which probably means it's created by yourself), you don't want to overwrite it since most likely you have another instance running - but we can't be sure if you simply forget to terminate it, or you intentionally let it run

Reply:
Updated it ✅